site/themes/s2b_hugo_theme/layouts/partials/cal/edit.html

```diff
@@ -472,6 +472,12 @@ <h2>
           </div>
         </div>
       </div>
+      [[/ id ]]
+
+      [[# id ]]
+      <input type="hidden" name="code_of_conduct" value="[[# codeOfConduct ]]1[[/ codeOfConduct ]]" />
```

Comment:
this looks good, but maybe it'd be worth it to instead change the server to only validate the two checkboxes for new events. 



having special case code on the client to send up a 1 value, for the validator to check and discard feels a bit like working around the issue of the overzealous validator


for example: 

https://github.com/shift-org/shift-docs/blob/5c0326fdd5cea62bea70159270971783f63c6b15/app/models/calEventValidator.js#L185 

```
if (!input.id) { 
  v.requireTrue('code_of_conduct', "You must have read the Ride Leading Comic");
  v.requireTrue('read_comic', "You must agree to the Code of Conduct");
}
```

( oh! and the server side messages are reversed too! )

Reply:
@carrythebanner  -- don't worry about this comment being a blocker or anything. 
not showing those checkboxes seems like a nice improvement.
we could worry about improving the roundtrip of those flags as part of some other more general issue about reducing the number of fields.

Reply:
(Finally coming back to this) Good call, I incorporated that change into the backend. The frontend still hides the terms section when it's not needed, but those hidden inputs are no longer present.